Dockerfile

```diff
@@ -103,7 +103,7 @@ CMD ["pnpm", "dev"]
 # - the official image we publish
 # - purposefully last so that it is the default target when building
 ####################################################################################################
-FROM ghcr.io/akuity/kargo-render:v0.1.0-rc.38 as final
+FROM ghcr.io/akuity/kargo-render:v0.1.0-rc.39 as final
```

Comment:
This fixes a bug where Kargo render sometimes pushed a new, empty target branch to the remote even when instructed _not_ to commit anything.